man/useradd.8.xml

```diff
@@ -180,11 +180,11 @@
 	</term>
 	<listitem>
 	  <para>
-	    The date on which the user account will be disabled. The date is
+	    Sets the date on which the user account will be disabled. The date must be
 	    specified in the format <emphasis remap='I'>YYYY-MM-DD</emphasis>.
```

Comment:
useradd and usermod accept the same input values as chage for `expiredate`. I'd recommend you to use the same text here.

Reply:
Thanks. Now it's updated